app.py

```diff
@@ -152,15 +152,19 @@ def session_stats_create(session_key, caller_identity):
     bytes_sent = payload.get('bytes_sent')
     bytes_received = payload.get('bytes_received')
     if bytes_sent < 0:
-        return jsonify({'error': 'bytes_sent should not be negative'}), 400
+        return jsonify(error='bytes_sent should not be negative'), 400
     if bytes_received < 0:
-        return jsonify({'error': 'bytes_received should not be negative'}), 400
+        return jsonify(error='bytes_received should not be negative'), 400
 
     session = Session.query.get(session_key)
     if session is None:
         session = Session(session_key)
         session.client_ip = request.remote_addr
```

Comment:
Would be worth to sync names:
- `client_id` + `client_ip`
- `consumer_id` + `consumer_ip`

Reply:
Same applies for other model fields as well: `client_bytes_sent`, `client_bytes_received`, `node_bytes_sent`, `node_bytes_received`, `client_updated_at`, `node_updated_at`, `node_key`.
Lets not solve all problems at once.